### PR TITLE
Add data-section-label attribute to dashboard nav groups

### PR DESCRIPTION
**Pull request by webteckie**
And use explicit props (vs. spread operator) in previous changes.

### CODE REVIEW

File: admin/client/views/home.js [hunk 1]
```diff
@@ -124,11 +124,12 @@ var HomeView = React.createClass({
...
 	renderOrphanedLists () {
 		if (!this.props.orphanedLists.length) return;
+		let sectionLabel = 'Other';
 		return (
 			<div className="dashboard-group">
-				<div className="dashboard-group__heading">
+				<div className="dashboard-group__heading" data-section-label={sectionLabel}>
 					<span className="dashboard-group__heading-icon octicon octicon-database" />
-					Other
+					{sectionLabel}
```
Comment (mxstbr) [inline admin/client/views/home.js]: Why was this changed?

Comment (webteckie) [inline admin/client/views/home.js]: because the label "Other" is now used is multiple(2) places...just abstracting to a local var for clarity

**Comment by mxstbr** [comment]
LGTM
